source/_integrations/buienradar.markdown

```diff
@@ -62,7 +62,7 @@ The following entities will be created:
 - **Wind azimuth**: Where the wind is coming from in degrees, with true north at 0° and progressing clockwise.
 - **Pressure**: "The sea-level air pressure in [hPa](https://en.wikipedia.org/wiki/Hectopascal)."
 - **Visibility**: "Visibility in meters ([m](https://en.wikipedia.org/wiki/Metre))."
-- **Wind gust**: "The wind speed of wind gusts ([m/s](https://en.wikipedia.org/wiki/M/s))."
+- **Wind gust**: "The wind speed of wind gusts ([km/h](https://en.wikipedia.org/wiki/M/s))."
```

Comment:
The link to Wikipedia is now incorrect, could you please adjust that too?

Thanks! 👍